api/src/main/java/org/apache/cloudstack/api/command/admin/offering/CreateDiskOfferingCmd.java

```diff
@@ -144,6 +144,9 @@
             description = "Hypervisor snapshot reserve space as a percent of a volume (for managed storage using Xen or VMware)")
     private Integer hypervisorSnapshotReserve;
 
+    @Parameter(name = ApiConstants.CACHE_MODE, type = CommandType.STRING, required = false, description = "the cache mode to use for this disk offering. none, writeback or writethrough")
```

Comment:
@radu-todirica why no since attribute here? no biggy, just wondering if you forgot.